config/config.go

```diff
@@ -152,7 +152,7 @@ func (cfg *config) Get(key string, encryptionKey string) (string, error) {
 
 func (cfg *config) get(key string, encryptionKey string, gormDB *gorm.DB) (string, error) {
 	var userConfig db.UserConfig
-	err := gormDB.Where(&db.UserConfig{Key: key}).Limit(1).Find(&userConfig).Error
+	err := gormDB.Where("key = ?", key).First(&userConfig).Error
```

Comment:
`First` returns an error if the key is not found, which will lead us to return errors in a lot of places the code currently does not expect.